src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_commands.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, method_name):
             method_name, recording_processors=[KeyReplacer()]
         )
 
+    @skip('Live run succeeded, recording test: DecodeError')
```

Comment:
Could you please show me the specific error?

Reply:
OK, it seems that this issue is not related to the bumping version, there may be a problem with this test itself~
@vilit1 Could you please help review and approve the `Iot` module related code? And could you help submit a separate PR to help solve the problem of this failed test?

Reply:
I have a comment describing the issue with re-recording this test below the definition
`# for some reason the recording is missing a ] and a } after the routing.endpoints.eventHubs`

this requires manual fixing

Reply:
OK, thanks for your help. Because this issue is not related to bumping version, we will fix it manually first. Please help submit a PR separately to fix the test issue in the future, thanks~